traveler-swift-core/TravelerKit/Delegates/WishlistRemoveDelegate.swift

```diff
@@ -10,16 +10,17 @@ import Foundation
 /// Notified when removing an item from the wishlist
 public protocol WishlistRemoveDelegate: class {
     /**
-     Called when the `CatalogItem` was removed from the wishlist successfully
+     Called when the `Product` was removed from the wishlist successfully
      - Parameters:
-     -item: The `CatalogItem`that was removed from the wishlist
+     - itemDetails: The corresponding `CatalogItemDetails` of the `Product` that was removed from the wishlist. If the removed `Product` is unavailable, `item` will be nil becuase there's no correspoding `CatalogItemDetails` of a unavailable `Product`.
      */
-    func wishlistRemoveDidSucceed()
+    func wishlistRemoveDidSucceedFor(_ itemDetails: CatalogItemDetails?)
```

Comment:
See comment on refactoring this function signature on the view controller that implements it.